spring-cloud-dataflow-docs/src/main/asciidoc/using-spring-cloud-stream-modules.adoc

```diff
@@ -502,6 +768,25 @@ bucketcount:: A number of buckets
 A path part prefixed with a hash key i.e. `XXX_hash`.
 
 
+[[spring-cloud-stream-modules-jdbc]]
+=== JDBC (`jdbc`)
+
+A module that writes its incoming payload to an RDBMS using JDBC.
+
+==== Options
+
+The **$$jdbc$$** $$sink$$ has the following options:
+
+$$expression$$:: $$a SpEL expression used to transform messages$$ *($$String$$, default: ``)*
+$$tableName$$:: $$String$$ *($$String$$, default: `<stream name`)*
+$$columns$$:: $$the names of the columns that shall receive data, as a set of column[:SpEL] mappings, also used at initialization time to issue the DDL$$ *($$String$$, default: `payload`)*
+$$initialize$$:: $$String$$ *($$Boolean$$, default: `false`)*
+$$batchSize$$:: $$String$$ *($$long$$, default: `10000`)*
+
+The module also uses Spring Boot's http://docs.spring.io/spring-boot/docs/1.3.0.BUILD-SNAPSHOT/reference/htmlsingle/#boot-features-configure-datasource[DataSource support]
```

Comment:
IT would be better do not refer to `BUILD-SNAPSHOT`s from the Docs at all.
So, the best link here is:
http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-sql.html#boot-features-configure-datasource
Note: I don't refer to the `htmlsingle` as well. It is so big to wait for the particular page to be loaded...